### DISCUSSION TITLE
I think i just got banned for using this

### DISCUSSION THREAD

**Discussion by throwaway-cmd** [UltimaHoarder/UltimaScraper#1332] [General]
I got logged out I can no longer log back in.

login says wrong username/password (I know it's correct, it's in my password manager)
`{"error":{"code":0,"message":"Wrong email or password"}}` from /api2/v2/users/login
click on forgot password don't receive an email
`{"success":true}` /api2/v2/users/forgot-password but never receive email

**Comment by throwaway-cmd** [comment]
Confirmed by OF.

```
We regret to inform you that your account and all the associated information have been erased from our system. Accounts are typically being deleted due to suspicious/fraudulent activity, issuing a chargeback claim, creating multiple profiles or due to violating the Acceptable Use policy in any other way. Unfortunately, we are not able to override this automatic process. Please see more details in the 4th paragraph of our Terms.

Support Updates
OnlyFans
```

[Acceptable Use](https://onlyfans.com/terms#acceptable-use-policy)

**Comment by ghost** [comment]
The 4th paragraph of the Terms is:

> How we may change the Terms of Service: We may change any part of the Terms of Service without telling you beforehand in the following circumstances:

> // This is for fans and creators
> Other terms which may apply to your use of OnlyFans: The following additional terms may apply to your use of OnlyFans:

> Do not use OnlyFans in any way which may exploit, harm, or attempt to exploit or harm any individual under 18 years old, for example by exposing them to inappropriate Content.

> The rules of the OnlyFans Referral Program:

> Who can use this Complaints Policy? Whether or not you are a User of OnlyFans, you can use this Complaints Policy to alert us to any complaint which you have relating to OnlyFans.

> What do these Platform to Business Regulation Terms include? These Platform to Business Regulation Terms provide information about our practices to comply with the fairness and transparency requirements set out in EU Regulation 2019/1150 (the "Platform to Business Regulation").

None of these terms would apply to using this software. The only paragraph that would come close is 19:

> Do not use any automated program, tool or process (such as web crawlers, robots, bots spiders, and automated scripts) to access OnlyFans or any server, network or system associated with OnlyFans, or to extract, scrape, collect, harvest or gather Content or information from OnlyFans.

At the moment I would say correlation is not causation and you must've done something else, as was stated in the mail.

**Comment by OFfriend** [comment]
> suspicious/fraudulent activity, issuing a chargeback claim

OnlyFans hates chargeback claims because they cost them money. If you do this a lot, your account will surely be blocked. Or OnlyFans thinks you are not 18 years old.

**Comment by ghost** [comment]
How can I use this script on android? Anybody make an app yet and wanna
share? Or can someone send me the pics of just one person ill send the name
in personal message if so.
Thanks 😊 much love

**Comment by throwaway-cmd** [comment]
Just to be clear...

I am 30
I have never done a charge back
I do have a second account that is still active (never used this script on)
Have no idea how you could do 'fraudulent activity' as  'fan'

I hadn't been on OF for a few days. Logged in, looked through my messages,
went to buy something, got logged out with a blank message. Was no longer
able to log back in.

**Comment by ghost** [comment]
@lucas69012 

Create you own topic and don't hijack this one....

@throwaway-cmd 

> I do have a second account that is still active (never used this script on)

And this is what OF wrote:

.....creating multiple profiles......

There is the reason for the ban. 

> I hadn't been on OF for a few days. Logged in, looked through my messages, went to buy something, got logged out with a blank message. Was no longer able to log back in.

I assume that this would also happen on your second account.

**Comment by throwaway-cmd** [comment]
I still have access to the other account... so I dont think its that.

**Comment by mediaburnwayne** [comment]
So has anyone else had an issue with this? That would be the way to correlate this

**Comment by throwaway-cmd** [comment]
> So has anyone else had an issue with this? That would be the way to correlate this

Trying to get more info out of them, said that that's a bit generic I'd like to know what they are accusing me of.

**Comment by UltimaHoarder** [comment]
You didn't get banned for using the script. It's most likely due to your alt account.

**Comment by purpleemoji** [comment]
Hmmm. I have multiple accounts as well. I've used the script on all of them multiple times. Hope I'm safe.

**Reply by ghost** [reply]
I'd say we will have to wait what Onlyfans says to OP what the exact reason is.

**Comment by ghost** [comment]
@throwaway-cmd 

Has OF repsonded to your question?

**Comment by throwaway-cmd** [comment]
They gave some excuse about it being an "error with the system" and reinstated my accounts
